cprover_bindings/src/goto_program/symtab_transformer/transformer.rs

```diff
@@ -58,6 +58,7 @@ pub trait Transformer: Sized {
             Type::Signedbv { width } => self.transform_type_signedbv(width),
             Type::Struct { tag, components } => self.transform_type_struct(*tag, components),
             Type::StructTag(tag) => self.transform_type_struct_tag(*tag),
+            Type::TypeDef { name: tag, typ } => self.transform_type_def(*tag, typ),
```

Comment:
Sorry, not sure I understand the question. Are you wondering why I use `name` inside `typedef`?

If that's the case, it's because this doesn't actually map to a `StructTag`.